cypress/e2e/article.cy.js

```diff
@@ -1,24 +1,121 @@
 /// <reference types="cypress" />
 /// <reference types="../support" />
 
-describe('Article', () => {
-  before(() => {
-
-  });
+import { generateNewArticle } from "../support/generateNewArticle";
+import faker from 'faker';
 
+describe(' User is able to create an article', () => {
   beforeEach(() => {
     cy.task('db:clear');
+    cy.registerNewUser().then(user => {
+      cy.login(user).then(() => user);
+    }).as('user');
+
+    cy.visit(`#/editor`);
   });
 
   it('should be created using New Article form', () => {
+    const { articleTitle, articleDescription, articleBody, articleTags }
+      = generateNewArticle();
 
+    cy.getByDataCy('article-title')
+      .type(articleTitle);
+    cy.getByDataCy('article-description')
+      .type(articleDescription);
+    cy.getByDataCy('article-body')
+      .type(articleBody);
+      cy.getByDataCy('article-tags')
+      .first()
+      .type(`${articleTags}{enter}`);    
+    cy.getByDataCy('publish-btn').click();
+   // cy.wait(2000);
+    cy.url(`#/articles/${articleTitle}-`);
```

Comment:
The `cy.url` function should be used with `.should` like `cy.url().should('include', '/articles')`